src/_guides/language/language-tour.md

```diff
@@ -716,6 +716,68 @@ var constantList = const [1, 2, 3];
 // constantList[1] = 1; // Uncommenting this causes an error.
 {% endprettify %}
 
+<a id="spread-operator"> </a>
+Dart 2.3 introduced the **spread operator** (`...`) and the
+**null-aware spread operator** (`...?`),
+which provide a concise way to insert multiple elements into a collection.
+
+For example, you can use the spread operator (`...`) to insert
+all the elements of a list into another list:
+
+<?code-excerpt "misc/test/language_tour/built_in_types_test.dart (list-spread)"?>
+{% prettify dart %}
+var list = [1, 2, 3];
+var list2 = [0, ...list];
+assert(list2.length == 4);
+{% endprettify %}
+
+If the expression to the right of the spread operator might be null,
+you can avoid exceptions by using a null-aware spread operator (`...?`):
+
+<?code-excerpt "misc/test/language_tour/built_in_types_test.dart (list-null-spread)"?>
+{% prettify dart %}
+var list;
+var list2 = [0, ...?list];
+assert(list2.length == 1);
+{% endprettify %}
+
+For more details and examples of using the spread operator, see the 
+[spread operator proposal.][spread proposal]
+
+<a id="collection-operators"> </a>
+Dart 2.3 also introduced **collection if** and **collection for**,
+which you can use to build collections using conditionals (`if`)
+and repetition (`for`).
+
+Here's an example of using **collection if**
+to create a list with zero, one, or two items in it:
```

Comment:
This is in the list section, so every example is about lists.

Reply:
I still don't get it. The text says "to create a list with zero, one, or two items in it" but the only three variable being initialized contain a bool, a bool, and a string. None of them are lists.

Reply:
The following code creates the list containing 0-2 strings.

Reply:
It's the `nav` variable in the [example that follows that text](https://kw-www-dartlang-1.firebaseapp.com/guides/language/language-tour#collection-operators). I'll change the description to be clearer. I think I'll remove the bools from the example, too... I dithered about whether they were helpful. Clearly, they aren't!

Reply:
Oh, I hadn't spotted that this was a list. I think it might be a lot easier to read if there were some non-conditional elements in the list also. For example:

`var nav = ['Home', 'Furniture', 'Plants', 'Lamps', if (promoActive) 'Outlet']`

Reply:
Oh, nice idea. And more realistic than my example.